models/customers-search-list-my-view/src/types.ts

```diff
@@ -0,0 +1,21 @@
+import { TLocalizedStringGraphql } from '@commercetools-test-data/commons';
+import { TBuilder } from '@commercetools-test-data/core';
+import type { TMcSettingsCustomersSearchListMyView } from '@commercetools-test-data/graphql-types';
+
+export type TCustomersSearchListMyView =
+  TMcSettingsCustomersSearchListMyView & {
+    name: string;
+  };
+
+export type TCustomersSearchListMyViewGraphql = Omit<
+  TCustomersSearchListMyView,
+  'nameAllLocales'
+> & {
+  nameAllLocales: TLocalizedStringGraphql | null;
```

Comment:
Do we actually need to overwrite this property?
I'm asking because the type in the `TCustomersSearchListMyView` is already a `LocalizedField`, which is what the Graphql API will return.

Also, in the original type it's defined as an array.

Reply:
Done https://github.com/commercetools/test-data/pull/654/commits/29a5a7e24f2bea40ab932efc3ada8db4b9edb70f